pkg/dao/repository.go

```diff
@@ -79,3 +114,12 @@ func ApiFieldsToModel(apiRepo *api.RepositoryRequest, repoConfig *models.Reposit
 		repoConfig.Version = *apiRepo.DistributionVersion
 	}
 }
+
+// Converts the database models to our response objects
+func convertToResponses(repoConfigs []models.RepositoryConfiguration) []api.RepositoryResponse {
+	repos := make([]api.RepositoryResponse, len(repoConfigs))
+	for i := 0; i < len(repoConfigs); i++ {
+		repos[i].FromRepositoryConfiguration(repoConfigs[i])
```

Comment:
Could you elaborate on this? I'm not sure what you mean

Reply:
sure!  so the api/repository.go  file requires the 'model' package:  https://github.com/content-services/content-sources-backend/blob/main/pkg/api/repository.go#L3

the reason for that is the FromRepositoryConfiguration() method.   Since we basically have the reverse in this file (ApiFieldsToModel), maybe it makes sense to have FromRepositoryConfiguration) live here, maybe named  ModelFieldsToApi ;)

Then the api package wouldn't depend on the model package at all

Reply:
makes sense! i'll move that